url_test.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
```

Comment:
![yas](https://user-images.githubusercontent.com/15213246/27218669-975ad3d0-529c-11e7-9e71-28472917353d.png)
URL has some issues. Correct it.